react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -149,6 +150,31 @@ public void editorDidMount(boolean hasUnsupportedBlocks) {
                     setFocusOnTitle();
                 }
             }
+
+            @Override
+            public void debug(String message) {
+                AppLog.d(AppLog.T.EDITOR, message);
```

Comment:
Discussion points: Do we need to add new Enum case for `AppLog.T.EDITOR`?

Reply:
Good question, and thanks for opening the things that popped up in your mind and frame them as discussion points here. As to the specific question, I think it's ok to have a new enum to make a distinction between what happens on JS and what not, but maybe better try as is and see if we need one first? I don't have a strong opinion in this regard I think

Reply:
Thanks! looks like `EDITOR` is used in other modules so may be better to have new one😄